lib/mayaHydra/hydraExtensions/mayaHydraParams.h

```diff
@@ -39,6 +39,7 @@ struct MayaHydraParams
     float motionSampleStart = 0;
     float motionSampleEnd = 0;
     bool  displaySmoothMeshes = true;
+    int   refineLevel = 0;
```

Comment:
What about the refineLevel which is adapters, MayaHydraAdapter::GetDisplayStyle().refineLevel > 0)
Is what you do replaces this ? Or is this something else ?

Reply:
I assume the added one is the global refine level, while the one in DisplayStyle is per-primitive, but it's not enabled yet due to MRenderItem was already refined. HdsiLegacyDisplayStyleOverrideSceneIndex can be used to override the per-primitive one.

Reply:
As far as I understand MayaHydraAdapter is used only for Maya native prims. The new scene index is supposed to be used only for USD prims. I will add "addExcludedSceneRoot" to it in my next commit to exclude Maya prims